runtime/common/tests/xcm_mock/parachain.rs

```diff
@@ -694,6 +703,11 @@ pub(crate) fn create_asset_metadata(
 }
 
 pub(crate) fn create_asset_location(parents: u8, para_id: u32) -> AssetLocation {
+    if parents == 0 {
+        return AssetLocation(VersionedMultiLocation::V1(MultiLocation::new(
+            parents, Here,
+        )));
+    }
     AssetLocation(VersionedMultiLocation::V1(MultiLocation::new(
         parents,
```

Comment:
Maybe it's a stupid question.

Does the caller check `parents`?
Basically, it should be `0` or `1`.

Reply:
Yeah should be 0 or 1 for now, added a panic otherwise.